dev/com.ibm.ws.concurrent_fat_reactive/test-applications/reactiveapp/src/reactiveapp/web/ConcurrentReactiveServlet.java

```diff
@@ -182,4 +183,25 @@ public void contextualizedFlowTest() throws Exception {
         }
     }
 
+    /**
+     * Test that context can be accessed in an Operator and a Flow.Subscriber subscribed to a Multi
+     *
+     */
+    @Test
+    public void mutinyOperatorsTest() throws Exception {
+        final ContextCDLImpl continueLatch = new ContextCDLImpl(2);
+
+        Multi.createFrom().item(continueLatch).emitOn(executor).invoke(cdl -> {
+            try {
+                cdl.checkContext();
+                cdl.countDown();
+            } catch (NamingException e) {
+                throw new RuntimeException(e);
+            }
+        }).subscribe().withSubscriber(new ThreadSubscriber());
+
+        if (!continueLatch.await(TIMEOUT_NS, TimeUnit.NANOSECONDS))
+            throw new AssertionError("Timed out waiting for CountDownLatch, context may not have been available");
```

Comment:
I'd recommend preserving this exception and chaining it to the AssertionError if possible.